tools/topology/topology1/m4/drc_coef_threshold_-25_knee_15_ratio_10_post_3.m4

```diff
@@ -0,0 +1,21 @@
+# Exported Control Bytes 10-Nov-2023
```

Comment:
can we add comments for this blob usage here?
maybe filename not need so complicated, we can add its usage and specification here.

Reply:
A generic name like drc_coef_generic_notebook_speaker.m4 / .conf might do after I can decide what are the final parameters to use.